documentation/book/assembly-kafka-bridge-quickstart.adoc

```diff
@@ -0,0 +1,57 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-the-kafka-bridge.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+:parent-context-using-the-kafka-bridge: {context}
+:context: {parent-context-using-the-kafka-bridge}-bridge-quickstart
+
+[id='assembly-kafka-bridge-quickstart-{context}']
+= Kafka Bridge quick start
+
+Use this quick start to try out the {ProductName} Kafka Bridge in your local development environment. You will learn how to:
+
+* Deploy the Kafka Bridge to your OpenShift cluster
+* Expose the Kafka Bridge Service to your local machine
+* Produce messages to topics, including to a specific partition
+* Create a Kafka Bridge consumer for use in this quick start
+* Perform basic consumer operations
+
+HTTP requests are formatted as curl commands that you can copy and paste to your terminal. You can use an API testing tool to call the API if preferred. As described in xref:con-requests-kafka-bridge-str[Requests to the {ProductName} Kafka Bridge], all POST requests used in this quick start supply the following Content-Type header:
+
+[source,http,subs=attributes+]
+----
+Content-Type: application/vnd.kafka.v2+json
+----
```

Comment:
it depends ... if you are describing how to do POST for sending messages and you are using JSON as payload in your examples, then the content type is `application/vnd.kafka.json.v2+json`

Reply:
Excellent point. I'll update this content and ask you to re-check it.

Also, I think "Creating a Kafka Bridge consumer" should explain _why_ we are using a Content-Type header of `application/vnd.kafka.v2+json`.

Reply:
Please check the updated information when you get a moment, thank you.